open-metadata-implementation/access-services/subject-area/subject-area-api/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/responses/SubjectAreaOMASAPIResponse.java

```diff
@@ -20,26 +21,23 @@
  */
 @JsonAutoDetect(getterVisibility = PUBLIC_ONLY, setterVisibility = PUBLIC_ONLY, fieldVisibility = NONE)
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonIgnoreProperties(ignoreUnknown = true)
-public class SubjectAreaOMASAPIResponse<R> extends FFDCResponseBase {
+@JsonIgnoreProperties(ignoreUnknown = true, value = {"head"})
```

Comment:
I am not sure what `value = {"head"})` is doing here - I am not understanding; does this relate to the getHead methods?  Please could you add a comment to explain why it is needed and how it impacts the other parts of the code.

Reply:
Yes, `value = {"head"}`means that Jackson does not serialize result from getter (getHead). Because it does not need for clients in their json.

Reply:
Ah ok thanks.